doc/UsersGuide/source/CodeOverview.rst

```diff
@@ -81,22 +94,33 @@ When the ufs-weather-model is cloned, the basic directory structure will be simi
    │   │   ├── physics      -------- CCPP compliant physics schemes
    │   │   └── suites       -------- CCPP physics suite definition files (SDFs)
    │   ├── cpl              -------- Coupling field data structures
-   │   ├── gfsphysics
-   │   │   ├── CCPP_layer
-   │   │   ├── GFS_layer
-   │   │   └── physics     --------- unused - IPD version of physics codes
-   │   ├── io              --------- FV3 write grid comp code
-   │   ├── ipd             --------- unused - IPD driver/interfaces
-   |   ├── stochastic_physics  ----- Cmakefile for stochastic physics code
-   ├── log                 --------- log files from NEMS compset regression tests
-   ├── modulefiles         --------- system module files for supported HPC systems
-   ├── NEMS                --------- NOAA Earth Modeling System framework
-   │   ├── exe
-   │   ├── src
-   │   └── test
-   ├── parm                --------- regression test configurations
+   │   ├── io               -------- UFSAtm write grid comp code
```

Comment:
can we add "ccpp" directory here?

Reply:
you want to move the tree structure for ccpp down so that it comes after io?

Reply:
Sorry, I missed that ccpp is in the structure.